skopeo.spec

```diff
@@ -204,7 +216,7 @@ sort -u -o devel.file-list devel.file-list
 %if ! 0%{?with_bundled}
 export GOPATH=%{buildroot}/%{gopath}:%{gopath}
 %else
-export GOPATH=%{buildroot}/%{gopath}:$(pwd)/Godeps/_workspace:%{gopath}
+export GOPATH=%{buildroot}/%{gopath}:$(pwd)/vendor/_workspace:%{gopath}
```

Comment:
You are right. "/_workspace" must be deleted. Did not noticed that when rewritting the expression. As tests are not run, this has no effect on the final rpms.